core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -22,65 +22,64 @@ var action = require('./action.js');
 var forms = require('./forms.js');
 var waitFor = require('./waitFor.js');
 var interactions = require('../../../extensions/interactions/protractor.js');
-var action = require('./action.js');
```

Comment:
Why removed and why no error?

Reply:
I have not migrated that part of ExplorationPlayerPage which uses action as it was not needed for these four suites, hence it was giving lint error as the variable declared was not used. So I removed it in order to fix lint checks.

Reply:
It was declared twice, at line 25 and also at line 21. So I removed one declaration